eZ/Publish/API/Repository/Tests/SectionServiceTest.php

```diff
@@ -218,14 +267,71 @@ public function testUpdateSection()
         /* END: Use Case */
 
         // Verify that service returns an instance of Section
-        $this->assertInstanceOf('\\eZ\\Publish\\API\\Repository\\Values\\Content\\Section', $updatedSection);
+        $this->assertInstanceOf(Section::class, $updatedSection);
 
         // Verify that the service also persists the changes
         $updatedSection = $sectionService->loadSection($standardSectionId);
 
         $this->assertEquals('New section name', $updatedSection->name);
     }
 
+    /**
+     * Test for the updateSection() method.
+     *
+     * @see \eZ\Publish\API\Repository\SectionService::updateSection()
+     * @depends eZ\Publish\API\Repository\Tests\SectionServiceTest::testCreateSection
+     * @depends eZ\Publish\API\Repository\Tests\SectionServiceTest::testLoadSection
+     * @depends eZ\Publish\API\Repository\Tests\SectionServiceTest::testNewSectionUpdateStruct
```

Comment:
TBH I feel like if test does not require `@depends` test result (return value), we should skip that tag. Otherwise it results in a need to provide a chain of test methods names when trying to execute just a single test, so it's bad DX for me.